### PR TITLE
📁 Fix flow export results to include results from replaced rulesets

### PR DESCRIPTION
**Pull request by norkans7**
Adresses https://github.com/rapidpro/rapidpro/issues/879

### CODE REVIEW

File: temba/flows/models.py [hunk 1]
```diff
@@ -4888,6 +4890,8 @@ def _write_runs(
             result_values = []
             for n, node in enumerate(result_nodes):
                 node_result = results_by_node.get(node.uuid, {})
+                if not node_result:
+                    node_result = results_by_name.get(node.label, {})
```
Comment (rowanseymour) [inline temba/flows/models.py]: not getting why we need the look up by node uuid? I think it's enough to just get the unique ruleset labels, and then just find values that match those labels

Comment (norkans7) [reply temba/flows/models.py]: my guess is that it is to do the right thing if we export many flows at the same time with similar ruleset labels

Comment (norkans7) [reply temba/flows/models.py]: Added an extra check for that in https://github.com/nyaruka/rapidpro/pull/2333/commits/1306dd69b4d968c3b2f5d1a004ba8b46e13bc50a

Comment (rowanseymour) [reply temba/flows/models.py]: Yeah so with that change, I think you can drop the initial lookup by node

File: temba/flows/models.py [hunk 2]
```diff
@@ -4861,9 +4861,9 @@ def _write_runs(
             # get this run's results by node UUID
```
Comment (nicpottier) [inline temba/flows/models.py]: update comment

**Pull request merged**